[2025-01-10T17:11:04.406+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T17:11:04.420+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:10:00+00:00 [queued]>
[2025-01-10T17:11:04.427+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:10:00+00:00 [queued]>
[2025-01-10T17:11:04.429+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T17:11:04.441+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 17:10:00+00:00
[2025-01-10T17:11:04.447+0000] {standard_task_runner.py:72} INFO - Started process 1132 to run task
[2025-01-10T17:11:04.449+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T17:10:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmp7mzh47wh']
[2025-01-10T17:11:04.452+0000] {standard_task_runner.py:105} INFO - Job 135: Subtask append_to_database
[2025-01-10T17:11:04.492+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:10:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T17:11:04.566+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T17:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T17:10:00+00:00'
[2025-01-10T17:11:04.568+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T17:11:20.355+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T17:11:20.356+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T17:11:20.368+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T17:11:20.374+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T17:10:00+00:00, execution_date=20250110T171000, start_date=20250110T171104, end_date=20250110T171120
[2025-01-10T17:11:20.431+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T17:11:20.454+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T17:11:20.461+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
